lib/components/result-view/history.js

```diff
@@ -56,7 +56,7 @@ const History = observer(({ store }: { store: OutputStore }) => {
           />
         </div>
       </div>
-    : null;
+    : <ul className="background-message centered">No output to display</ul>;
```

Comment:
The reason I used `<ul>` was so the atom styling for `className="background-message centered"` would work (it doesnt if I just wrap in a `<div>`). Is it better if I wrap the ul tag in a `<div>`?

![styleguide](https://user-images.githubusercontent.com/10860657/27016990-7ffa4000-4eea-11e7-8be1-cd804d748481.png)